rtos/TARGET_CORTEX/rtx5/RTX/Source/rtx_lib.c

```diff
@@ -122,9 +122,12 @@ static osRtxThread_t os_idle_thread_cb \
 __attribute__((section(".bss.os.thread.cb")));
 
 // Idle Thread Stack
+#if defined (__ARMCC_VERSION)
```

Comment:
Two things:
* I'm assuming that checks whether it's 5 or 6, but it's not obvious which way, could you add comment? Also why are we only doing it for one compiler should be documented.
* Is this commit accepted into CMSIS repo? If so please reference it in the commit message. I really want to avoid modifying RTX/CMSIS in Mbed.